docs/fundamentals/code-analysis/quality-rules/snippets/csharp/all-rules/ca1002.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+
+namespace ca1001
+{
+    //<snippet1>
+    // This class violates the rule.
+    public class MutableItems
+    {
+        // CA1002: Change 'List<string>' in 'ViolationItems.Members' to use 'Collection<T>', 'ReadOnlyCollection<T>' or 'KeyedCollection<K,V>'
```

Comment:
Is 'ViolationItems.Members' coming from the error message? Should it say 'MutableItems.Items'?